backend/src/v2/test/Makefile

```diff
@@ -9,19 +9,9 @@ include $(ENV_PATH)
 SHELL = /bin/bash
 
 .PHONY: integration-test
-integration-test: upload
+integration-test:
 	export KF_PIPELINES_ENDPOINT=$(HOST) \
-		&& python -u sample_test.py \
-		--samples_config samples/test/config-integration.yaml \
-		--timeout_mins 60 \
-		--context $(GCS_ROOT)/src/context.tar.gz \
-		--gcs_root $(GCS_ROOT)/data \
-		--gcr_root $(GCR_ROOT) \
-		--kfp_package_path "$(KFP_PACKAGE_PATH)"
-
-.PHONY: upload
-upload: context
-	python -u scripts/upload_gcs_blob.py tmp/context.tar.gz $(GCS_ROOT)/src/context.tar.gz
+		&& python -u ../../../../samples/v2/sample_test.py
```

Comment:
going off the diff here, can you not just leave this line alone (ie `python -u sample_test.py`)?  It appears to already be in that directory as sample_test.py only exists there

Reply:
It was changed because the path was wrong.

Reply:
updated: it appears the 'old' command may not be pointed correctly, or the `upload` Target run before the command somehow changes the working directory to a place where the script does exist.  tl;dr: not a problem